personal_profile/models.py

```diff
@@ -14,6 +14,6 @@ class PersonalProfile(models.Model):
     about= models.TextField() 
     birth_date= models.DateField()
     profile_pic= models.ImageField(null=True, blank=True)
-  
+
```

Comment:
What changes were made in here ?

Reply:
@kasemAlem the flask8 asked to have a newline at the end of file with no space